config/helpers.php

```diff
@@ -790,60 +790,87 @@ function tc(string $key, int $count)
  */
 function timestamp(?string $date = null, $step = null): ?int
 {
+    // only allow valid dates as input
     if (V::date($date) === false) {
         return null;
     }
 
+    // convert date to timestamp
     $date = strtotime($date);
```

Comment:
What happens if `$date` cannot be interpreted by `strtotime`?

Reply:
A few lines up `V::date($date)` takes care of it.

Reply:
@nilshoerrmann You may be right 🤔 

````php
$date = 'january';

var_dump(strtotime($date));
var_dump(date_parse($date));
````

**Output**

````
int(1611183600)

array(12) {
  ["year"]=>
  bool(false)
  ["month"]=>
  int(1)
  ["day"]=>
  bool(false)
  ["hour"]=>
  bool(false)
  ["minute"]=>
  bool(false)
  ["second"]=>
  bool(false)
  ["fraction"]=>
  bool(false)
  ["warning_count"]=>
  int(1)
  ["warnings"]=>
  array(1) {
    [8]=>
    string(27) "The parsed date was invalid"
  }
  ["error_count"]=>
  int(0)
  ["errors"]=>
  array(0) {
  }
  ["is_localtime"]=>
  bool(false)
}

````

Reply:
@afbora but that looks correct - we wouldn't want to allow just `january` as input and `V::date()` will not allow it - or am I reading this wrong?

Reply:
You read it right, I thought wrong 🙂 The both return different results, but `V::date()` works first.